rinja_derive/src/lib.rs

```diff
@@ -379,6 +382,40 @@ impl fmt::Display for MsgValidEscapers<'_> {
     }
 }
 
+#[derive(Debug)]
+struct OnceMap<K, V>([Mutex<HashMap<K, V, FxBuildHasher>>; 8]);
```

Comment:
If it comes from the `once_map` crate, can you add a comment saying so please?

Reply:
The API idea comes from `once_map`, but I copied no code. I did not look into their source to implement my struct, but I daresay their code is not much different. I would not call my code derivative work. Still, I added a notice of the original authorship of the API.

Reply:
Oh then doesn't matter. Impressive work!